.readme-partials.yml

```diff
@@ -132,3 +132,24 @@ body: |-
     [LOGGING_TRACE_KEY]: 'custom-trace-value'
   }, 'Bunyan log entry with custom trace field');
   ```
+
+  ### Error handling with a default callback
+  The `LoggingBunyan` class creates an instance of `Logging` which creates the `Log` class from `@google-cloud/logging` package to write log entries. 
+  The `Log` class writes logs asynchronously and there are cases when log entries cannot be written and an error is 
```

Comment:
nit: I suggest to give an example of the cases or just to say "...and throws the error when it fails or an error is returned from Logging backend".